index.bs

```diff
@@ -2443,74 +2325,116 @@ partial interface MLGraphBuilder {
   </div>
 </details>
 
-<details open>
+#### add #### {#api-mlgraphbuilder-binary-add}
+Add the values of the two input tensors, element-wise. See [[#api-mlgraphbuilder-binary]] for more detail.
+
+#### div #### {#api-mlgraphbuilder-binary-div}
+Divide the values of the first input tensor with the values of the second tensor, element-wise. See [[#api-mlgraphbuilder-binary]] for more detail.
+
+#### max #### {#api-mlgraphbuilder-binary-max}
+Select the greater values of the two input tensors, element-wise. See [[#api-mlgraphbuilder-binary]] for more detail.
+
+#### min #### {#api-mlgraphbuilder-binary-min}
+Select the lesser values of the two input tensors, element-wise. See [[#api-mlgraphbuilder-binary]] for more detail.
+
+#### mul #### {#api-mlgraphbuilder-binary-mul}
+Multiply the values of the two input tensors, element-wise. See [[#api-mlgraphbuilder-binary]] for more detail.
+
+#### pow #### {#api-mlgraphbuilder-binary-pow}
+Compute the values of the values of the first input tensor to the power of the values of the second input tensor, element-wise. See [[#api-mlgraphbuilder-binary]] for more detail.
+
+#### sub #### {#api-mlgraphbuilder-binary-sub}
+Subtract the values of the second input tensor from the values of the first input tensor, element-wise. See [[#api-mlgraphbuilder-binary]] for more detail.
+
+### Element-wise logical operations ### {#api-mlgraphbuilder-logical}
+Compare two input tensors element-wise, if the operation requires two inputs, and return a tensor of Boolean values 0 or 1 for the specified input values.
+
+The two input tensors will be broadcasted according to [[!numpy-broadcasting-rule]]. The [=rank=] of the output tensor is the maximum
+[=rank=] of the input tensors.
+
+<script type=idl>
+partial interface MLGraphBuilder {
+  MLOperand equal(MLOperand a, MLOperand b);
+  MLOperand greater(MLOperand a, MLOperand b);
+  MLOperand greaterOrEqual(MLOperand a, MLOperand b);
+  MLOperand lesser(MLOperand a, MLOperand b);
+  MLOperand lesserOrEqual(MLOperand a, MLOperand b);
+  MLOperand not(MLOperand a);
+};
+</script>
+
+<div>
+    **Arguments:**
+        - *a*: an {{MLOperand}}. The first input tensor.
+        - *b*: an {{MLOperand}}. The second input tensor.
+
+    **Returns:** an {{MLOperand}}. The output tensor that contains the result of
+    element-wise comparison of the two input tensors.
+</div>
+
+<details open algorithm>
   <summary>
-    The element-wise binary operation algorithms invoke the [=MLGraphBuilder/element-wise-binary-op | create element-wise binary operation=] steps as follows.
+    To <dfn for="MLGraphBuilder" data-lt="element-wise-logical-op">create element-wise logical operation</dfn> given |op|, |a| and |b|, run the following steps:
```

Comment:
It looks like this PR removes all references to the "create element-wise... operation" algorithms - and the IDL methods no longer have any definitions to link to. This seems like a step backwards.

Reply:
Fixed.

Reply:
Fixed in https://github.com/webmachinelearning/webnn/pull/478/commits/873eec1e656253ed5e5f600f0ebf7e5fd12b9746